api/v1/ytsaurus_types.go

```diff
@@ -811,7 +817,7 @@ type Component struct {
 }
 
 func (c Component) String() string {
-	return c.Name
+	return fmt.Sprintf("%s(%s)", c.Type, c.Name)
```

Comment:
This will look like ` updating components: [Master()]` for nameless componentns, let's add name only if not empty